include/net/ethernet.h

```diff
@@ -671,7 +671,16 @@ int net_eth_promisc_mode(struct net_if *iface, bool enable);
  * @return Pointer to PTP clock if found, NULL if not found or if this
  * ethernet interface does not support PTP.
  */
+#if defined(CONFIG_PTP_CLOCK)
```

Comment:
I have this modification already done as part of #16403

Reply:
ah yes, merge your PR I'll rebase